paperclip.gemspec

```diff
@@ -18,15 +18,15 @@ Gem::Specification.new do |s|
   s.require_paths = ["lib"]
 
   s.requirements << "ImageMagick"
-  s.required_ruby_version = ">= 1.9.2"
+  s.required_ruby_version = ">= 2.0.0"
 
-  s.add_dependency('activemodel', '>= 3.0.0')
-  s.add_dependency('activesupport', '>= 3.0.0')
+  s.add_dependency('activemodel', '>= 3.2.0', '>= 4.1.0')
```

Comment:
Just tried this out, and it doesn't work. The conditions are AND, and you can't install something that satisfies both here. I think we're going to have to stick with `>= 3.2.0`.